Makefile

```diff
@@ -215,7 +226,7 @@ test:: bin/gotestfmt
 	fi
 
 schemagen:: bin/${CODEGEN}
-schema: awsx/schema.json
+schema: .make/schema
```

Comment:
Do we need this sentinel even though the schema is just a single file?

Reply:
Yup - because it's a committed file, so sometime the timestamp might be updated during checkout, but it's not actually reflecting the newly checked-out code, so it should be re-generation but isn't due to the timestamp.

The short rule is: if it's committed or a directory, use a sentinel file.